### PR TITLE
Task/button large

### PR DESCRIPTION
**Pull request by JDMathew**
This adds large variant to the component library

- closes: #112

Aug-03-2022 19-39-58

### CODE REVIEW

**Comment by JDMathew** [comment]
Initially we decided on one prop to control both size and styling... we have changed tac to make this two props `variant` for the button type and `size` for the button size. Commit: [0bd63e2](https://github.com/functionland/apps-monorepo/pull/113/commits/0bd63e2edc3731d9e848b8a49216fe15bf5b6c0a) makes this change

File: libs/component-library/src/lib/theme/theme.ts [hunk 1]
```diff
@@ -93,111 +93,149 @@ const fxLightTheme = createTheme({
...
+      defaults: { ...textVariants.bodyXSSemibold },
+      large: { ...textVariants.bodySmallSemibold },
     },
   },
+};
+
+const fxLightTheme = createTheme({
+  ...BaseTheme,
+  textVariants,
+  buttonVariants: { ...buttonTheme.container.types },
```
Comment (keithluchtel) [inline libs/component-library/src/lib/theme/theme.ts:235]: with `buttonVariants`, do we want them to be able to pass in `disabled` and `pressed` when creating a button?

Comment (keithluchtel) [reply libs/component-library/src/lib/theme/theme.ts:235]: hmm, looks like the old implementation allowed these 2 options to be used as `buttonClasses` before

Comment (JDMathew) [reply libs/component-library/src/lib/theme/theme.ts:235]: Yeah the old version did allow it. Personally I don't think it's necessary to allow but doesn't hurt leaving it in

**Review by keithluchtel — APPROVED** [review]
🚀  thanks for making this change.

**Pull request merged**